src/store/store.ts

```diff
@@ -1309,7 +1309,7 @@ export const useStore = defineStore('store', {
       const isProprietor = this.getOrgPeople[0]?.roles[0]?.roleType === RoleTypes.PROPRIETOR
       const isOrganization = this.getOrgPeople[0]?.officer?.partyType === PartyTypes.ORGANIZATION
       const isDba = isProprietor && isOrganization
-      return !IsAuthorized(AuthorizedActions.FIRM_EDITABLE_DBA) && isDba
+      return IsAuthorized(AuthorizedActions.FIRM_EDITABLE_DBA) && isDba
```

Comment:
Small bug -- the meaning was changed. Previously this was "if not staff". Now it's "if authorized". Caught by the unit tests!